src/cljs/nr/gameboard.cljs

```diff
@@ -1375,6 +1374,20 @@
                                     (audio-sfx "run-successful")
                                     (audio-sfx "run-unsuccessful")
                                     (audio-sfx "virus-purge")))]
+        (r/create-class
+            {:display-name "audio-component"
+             :component-did-update
+             (fn []
+                 (update-audio {:sfx (:sfx @game-state) :sfx-current-id (:sfx-current-id @game-state)
```

Comment:
Could use `(select-keys @game-state [:sfx :sfx-current-id :gameid])` for this map.

Reply:
Cool; will update. :)

Reply:
Updated.